### ISSUE TITLE
Widgets.Slider admin : Clicking on thumbnail should open original image

### ISSUE THREAD

**Issue by dctremblay** [grandnode/grandnode#679]
When in list mode, click on the thumbnail :

- the link is the thumbnail itself instead of the original image
- it should be the original image.

When in edit mode, click on the thumbnail :

- there's no link at all
- it should link to the original image

**Comment by dctremblay** [comment]
I will assign it to intern

**Comment by shaykhsiddique** [comment]
![check](https://user-images.githubusercontent.com/18369069/70656775-edda7800-1c84-11ea-8458-983526db8e15.png)

The marked thumbnails links have problems ??

**Comment by dctremblay** [comment]
@shaykhsiddique Yep, if you click the image it should be the full scale image instead of the thumbnail.

**Comment by shaykhsiddique** [comment]
@dctremblay . I will fix it... How much USD will you provide for this ticket ??
